app/controllers/api/restful_controller.rb

```diff
@@ -95,10 +83,22 @@ def page_collection(collection)
     collection.offset(params[:from]).limit(params[:per] || default_page_size)
   end
 
+  def accessible_records
+    if current_user
+      visible_records
+    else
+      public_records
+    end
+  end
+
   def visible_records
     raise NotImplementedError.new
   end
 
+  def public_records
+    resource_class.visible_to_public.order(created_at: :desc)
```

Comment:
This is convenience more than anything else, but it works for groups, discussions, and motions (and could be easily adopted for comments too)